client/src/androidTest/java/io/getstream/chat/android/client/socket/TestChatSocketService.kt

```diff
@@ -1,48 +0,0 @@
-package io.getstream.chat.android.client.socket
-
-import io.getstream.chat.android.client.events.ChatEvent
-import io.getstream.chat.android.client.models.EventType
-import io.getstream.chat.android.client.parser.ChatParserImpl
-import io.getstream.chat.android.client.socket.okhttp.OkHttpSocketFactory
-import io.getstream.chat.android.client.token.TokenManagerImpl
-import org.junit.Test
-import java.lang.Thread.sleep
-
-class TestChatSocketService {
```

Comment:
Why did we remove this now?

Reply:
Because it didn't compile and was outdated. Seems these instrumentation tests weren't executed for some time